src/components/ButtonPanel.js

```diff
@@ -1,38 +1,43 @@
 import React from 'react';
+import PropTypes from 'prop-types';
 import Button from './Button';
 
-const ButtonPanel = () => (
+const ButtonPanel = ({ clickHandler }) => (
   <div>
     <div>
-      <Button name="AC" />
-      <Button name="+/-" />
-      <Button name="%" />
-      <Button name="÷" />
+      <Button handleClick={clickHandler} name="AC" />
+      <Button handleClick={clickHandler} name="+/-" />
+      <Button handleClick={clickHandler} name="%" />
+      <Button handleClick={clickHandler} name="÷" />
     </div>
     <div>
-      <Button name="7" />
-      <Button name="8" />
-      <Button name="9" />
-      <Button name="X" />
+      <Button handleClick={clickHandler} name="7" />
+      <Button handleClick={clickHandler} name="8" />
+      <Button handleClick={clickHandler} name="9" />
+      <Button handleClick={clickHandler} name="X" />
     </div>
     <div>
-      <Button name="4" />
-      <Button name="5" />
-      <Button name="6" />
-      <Button name="-" />
+      <Button handleClick={clickHandler} name="4" />
+      <Button handleClick={clickHandler} name="5" />
+      <Button handleClick={clickHandler} name="6" />
+      <Button handleClick={clickHandler} name="-" />
     </div>
     <div>
-      <Button name="1" />
-      <Button name="2" />
-      <Button name="3" />
-      <Button name="+" />
+      <Button handleClick={clickHandler} name="1" />
+      <Button handleClick={clickHandler} name="2" />
+      <Button handleClick={clickHandler} name="3" />
+      <Button handleClick={clickHandler} name="+" />
     </div>
     <div>
-      <Button name="0" />
-      <Button name="." />
-      <Button name="=" />
+      <Button handleClick={clickHandler} name="0" />
+      <Button handleClick={clickHandler} name="." />
+      <Button handleClick={clickHandler} name="=" />
     </div>
   </div>
 );
```

Comment:
- [ ] [Optional] it should implement `handleClick` function ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone5/panel/implementHandleClick.png))

```javascript
handleClick = (buttonName) => {
  this.props.clickHandler(buttonName);
};
```

**_this is only optional because it wasn't requested by the previous reviewer, but is originally a requirement, please take the time to add this simple function_**